orga/tests/acceptance/campaign-participants-individual-analysis-test.js

```diff
@@ -43,9 +42,10 @@ module('Acceptance | Campaign Participants Individual Analysis', function(hooks)
     assert.dom('[aria-label="Analyse par sujet"]').containsText('Sujets (2)');
   });
 
-  test('it should not display individual analysis when participation is not shared', async function(assert) {
+  test('it should not display individual analysis', async function(assert) {
```

Comment:
**question:** le test manque de contexte du coup. Pourquoi on affiche pas les données ?

Reply:
j'ai opté pour `it should not display individual analysis when tube recommendations are empty`